helpers/example/components/AppDocs.js

```diff
@@ -11,44 +11,50 @@ import { HomeDocs } from './HomeDocs';
 import { Page } from './Page';
 
 export function AppDocs({ components, packageName, pkg }) {
-	const [brand, setBrand] = useState('WBC');
+	const [brandName, setBrandName] = useState('WBC');
+
+	const brand = BRANDS[brandName];
 
 	return (
 		<Router>
 			<StrictMode>
-				<GEL brand={BRANDS[brand]}>
+				<Global
```

Comment:
Do all consumers now have to do this?

Reply:
Yeah these are just styles we want for our docs right?

The `body { margin: 0 }` reset style is mentioned in TODO conversation above. It's in our core reset already, but currently not used.

Reply:
Oh do you mean, do all consumers have to keep <Global /> outside of GEL? 

Our stylis plugin function in Core was prepending Global styles with `.GEL` (which we don't want). I may have just found a way to fix that, which will mean Global styles are ignored.

Reply:
That is now implemented. Styles passed to `<Global />` will not be processed (not prepended with the `.GEL ` parent class).